DataStructuresMemories/URLPresenterProtocol.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  URLPresenterProtocol.swift
+//  DataStructuresMemories
+//
+//  Created by Oleksii Kolomiiets on 4/20/18.
+//  Copyright © 2018 Oleksii Kolomiiets. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+protocol URLPresenterProtocol {
+    func setPresenter(by index: Int)
```

Comment:
"by" or "at"?

Reply:
"at" would be better choice. I'll fix it.